Dockerfile

```
@@ -1,9 +1,11 @@
-FROM alpine:latest
+FROM python:3.8-alpine
 
-RUN apk update && apk add python3 && mkdir -p /opt/amazon-mws-analytics
-COPY . /opt/amazon-mws-analytics
+WORKDIR /app
 
-WORKDIR /opt
-RUN cd /opt/amazon-mws-analytics && pip3 install -r requirements.txt
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
 
-ENTRYPOINT python3 -m amazon-mws-analytics
+COPY *.py ./
```

```suggestion
COPY *.py .
```
You don't need the trailing slash i don't think...

---

From [the docs](https://docs.docker.com/engine/reference/builder/#copy):

> If multiple `<src>` resources are specified, either directly or due to the use of a wildcard, then `<dest>` must be a directory, and it must end with a slash `/`.